core/modules/comment/templates/comment.tpl.php

```diff
@@ -60,12 +63,14 @@
 ?>
 <article class="<?php print implode(' ', $classes); ?> clearfix"<?php print backdrop_attributes($attributes); ?>>
 
-  <?php print render($title_prefix); ?>
   <?php if ($new): ?>
     <mark class="new"><?php print $new; ?></mark>
   <?php endif; ?>
-  <h3><?php print $title; ?></h3>
-  <?php print render($title_suffix); ?>
+  <?php if ($title_display == TRUE): ?>
+    <?php print render($title_prefix); ?>
+    <h3 class="comment-title"><?php print $title; ?></h3>
```

Comment:
I don't think we should be adding classes in here directly... Better to do it in the preprocess function so it can be overridden like all the other classes if need be.

Reply:
Right, that also came to my mind.

The preprocess function could create an array of classes and here in the template they get flattened like:

`class="<?php print implode(' ', $title_classes); ?>"`

Another option would be to create a complete attributes array and apply them here with:
`<?php print backdrop_attributes($title_attributes); ?>`
But that may be a little too much, as we don't need anything besides css class here (I guess).

I'm aware that there are more hardcoded css classes or attributes in templates, but personally, I find such arrays more flexible.

Reply:
@BWPanda and @indigoxela - I considered that, but didn't know how. This helps.